drivers/serial/uart_gecko.c

```diff
@@ -329,8 +329,13 @@ static void uart_gecko_config_func_1(struct device *dev);
 #endif
 
 static const struct uart_gecko_config uart_gecko_1_config = {
-	.base = UART1,
+#ifdef CONFIG_SOC_PART_NUMBER_EFM32HG322F64
```

Comment:
I think you can use "#ifdef CONFIG_SOC_SERIES_EFM32HG" instead of "#ifdef CONFIG_SOC_PART_NUMBER_EFM32HG322F64"

Reply:
I removed the this from the serial driver and reconfigured the board and soc to use usart1 instead of uart1.